### ISSUE TITLE
Metronome sounds

### ISSUE THREAD

**Issue by midenok** [MockbaTheBorg/MockbaMod#1]
Any chance to add more/replace metronome sounds?

**Comment by MockbaTheBorg** [comment]
I am not sure if it can be done. If those are sound files in the filesystem, then probably yes. 

Let's not use issues to ask questions. Let's direct questions either to the Facebook or discord.

**Comment by midenok** [comment]
Why? I would keep it open as there is a chance not only you will do the changes.

**Comment by MockbaTheBorg** [comment]
Issues are for break/fix actual issues.
Feature requests we deal with in the Facebook/Discord.
An issue open makes us come here expecting something is broken. :)

**Comment by intelliriffer** [comment]
> Any chance to add more/replace metronome sounds?

as advised, please join discord and we can discuss things, but answer is yes, I do it already. you can replace any existing metronome sound with your own sample. the file name though remains the same. For example I have cubase metronome sound assign to as Clap (clap.wav). you just replace factory sound with your own, on mockba mod since /usr is super mounted overlay, the change only happens in the overlay. keeping the root fs original (its read only anyways unless you mount it RW)

**Comment by midenok** [comment]
> An issue open makes us come here expecting something is broken. :)

You will get 90% of misses against your expectations. I think you should open discussions here, even if you don't look. Though what is the problem, I don't understand.

> but answer is yes

Nice, nice. Did you use aufs per chance? Any chance to expand the list?

> please join discord

Thanks, maybe later. Is discord google-indexed? I guess people who have similar problems search that in google.

**Comment by midenok** [comment]
@intelliriffer Is discord google-indexed?

**Comment by intelliriffer** [comment]
> @intelliriffer Is discord google-indexed?

Likely not.. and there is an as much case about not wanting these discussion indexed, as much case it is about wanting them.. 

Maybe Admin will enable github discussions on this repo. "Issues" still are about the problems about/around the content posted in this Repo. There are many other Forums that are active/google indexed discussing about Force/ Features and requests...

**Comment by midenok** [comment]
I mean this should be your interest to be indexed about feature requests/problems/solutions/discussions, etc. if you want to increase user flow. I found you by some inevident query, though I did many queries about various problems. I will open and close one more ticket, as I have to keep my ideas in one place, unless I moved them to somewhere else. Probably I will fork your repo and start creating issues at my place if you're getting irritated by unwanted tickets.

>  "Issues" still are about the problems about/around the content posted in this Repo. 

And there is Labels which are not enabled for posters here to mark issues as feature/bug/question/etc. Just in case you are unaware of this.
